crew

```diff
@@ -659,8 +659,21 @@ def resolve_dependencies
 
   return if @dependencies.empty?
 
+
   puts "The following packages also need to be installed: "
 
+  i = 0
+  last_deps = []
+  last_packages = ["curl", "gtk3", "sommelier"]
```

Comment:
I think we should find a better solution than hardcoding package names. I'm sure there are other packages affected by this that need a fix, too.

Reply:
Agreed this is not the best solution but it solves 3 known problems without a performance penalty with the dependency resolver routine.  If you can identify other packages, we can certainly add them here.  These are the issues I noticed so far.